pages/404.tsx

```diff
@@ -0,0 +1,31 @@
+import Image from "next/image";
+import Link from "next/link";
+
+const NotFound = () => {
+  return (
+    <main className=" min-h-screen flex px-4 flex-col-reverse justify-center bg-abeg-neutral-60 lg:flex-row">
```

Comment:
On mobile and tablet screens, the content of the page should be centered on the screen. It is currently starting at the top of the screen.